pkg/util/spanlogger/spanlogger.go

```diff
@@ -31,6 +31,20 @@ func New(ctx context.Context, method string, kvps ...interface{}) (*SpanLogger,
 	return logger, ctx
 }
 
+func FromContext(ctx context.Context) *SpanLogger {
```

Comment:
May you add a comment to this exported function, please?

Reply:
Shouldn't the linter get catch this ?